mandatory/3-playing-computer.js

```diff
@@ -40,3 +40,18 @@ for (let i = 0; i < 5; ++i) {
     console.log(e);
   }
 }
+// 1. This program throws an error. Why? (If you can't find it, try executing it).
+// x is not defined
```

Comment:
For questions 1 and 2, line 18 is actually fine as is. You're on the right tracks with something not being defined though! There's another line out there trying to use something that doesn't exist